### PR TITLE
Create your own Enumerable

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
- Create a class `MyList` that has an instance variable `@list`. ✅
- In `MyList` implement a method `#each` that yields successive members of `@list` and uses the `MyEnumerable` module. ✅
- Create a module `MyEnumerable` that implements the following methods (they should have the same funcionality as methods in [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html)):
   - `#all?` ✅
   - `#any?` ✅
   - `#filter` ✅
- Each class and module should has a separate .rb file. ✅
- Documented in a professional way. ✅

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,137 @@
...
+
+**I would like to thank to Microverse for giving this inspiring project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Bantealem) [inline README.md]: Great job so far.  currently, the `MIT` license is broken. kindly fix it.

**Review by Bantealem — CHANGES_REQUESTED** [review]
# Status: Change Required

 Hi @JuanDiegoGuerra and @lerfast 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Correct Git flow
- [x] Linters are passed
- [x] Descriptive PR summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (lerfast) [reply README.md]: Done!!!, thank you

image

**Review by skyv26 — APPROVED** [review]
## Hi @JuanDiegoGuerra and @lerfast,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Implement the features ✅ 
- Professional `README` template ✅ 
- Meaningful **Github** commits ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**